libs/execution_context/ebpf_general_helpers.c

```diff
@@ -224,8 +224,7 @@ ebpf_helper_function_prototype_t ebpf_core_helper_function_prototype_array[] = {
      BPF_FUNC_get_current_thread_create_time,
      "bpf_get_current_thread_create_time",
      EBPF_RETURN_TYPE_INTEGER,
-     {EBPF_ARGUMENT_TYPE_DONTCARE}}
-};
+     {EBPF_ARGUMENT_TYPE_DONTCARE}}};
```

Comment:
A bunch of these files have whitespace changes that seem unrelated to the rest of the PR.
Only change files that are needed for the issue fixed in the PR.
Changing whitespace globally should be done, if needed, in a separate PR.